src/Service/GithubEventService.php

```diff
@@ -218,38 +164,31 @@ public function importComponentsThroughRepo(ObjectEntity $repository, string $re
      */
     public function createRepository(Source $source, string $name, string $repositoryUrl): ?ObjectEntity
     {
+        $componentSchema  = $this->resourceService->getSchema($this->configuration['componentSchema'], 'open-catalogi/open-catalogi-bundle');
         $repositorySchema = $this->resourceService->getSchema($this->configuration['repositorySchema'], 'open-catalogi/open-catalogi-bundle');
         $mapping          = $this->resourceService->getMapping($this->configuration['repositoryMapping'], 'open-catalogi/open-catalogi-bundle');
 
-        // Get repository from github.
-        $repositoryArray = $this->githubApiService->getRepository($name, $source);
-        if ($repositoryArray === null) {
-            // Return error if repository is not found.
-            $this->data['response'] = new Response('Cannot find repository with url: '.$repositoryUrl, 404);
+        $this->configuration['organisationSchema']  = $this->configuration['organizationSchema'];
+        $this->configuration['organisationMapping'] = $this->configuration['organizationMapping'];
```

Comment:
Note: discussed this with @smisidjan, this is a temporary solution as things are in flux.